src/main/java/org/openrewrite/docker/table/DockerBaseImages.java

```diff
@@ -30,6 +30,14 @@ public DockerBaseImages(Recipe recipe) {
 
     @Value
     public static class Row {
+        @Column(displayName = "Source path before the run",
+                description = "The source path of the file before the run.")
+        String sourcePath;
+
+        @Column(displayName = "LST type",
+                description = "The LST model type that the file is parsed as.")
+        String type;
```

Comment:
The name and description hinted at possibly changing the file path, which isn't something we'll support.
```suggestion
        @Column(displayName = "Source path",
                description = "The source file containing the image reference.")
        String sourcePath;
```

The LST type I'd consider an implementation detail, that might still change in the future when we add new parsers. I'd lean towards leaving that out.